### DISCUSSION TITLE
Blade components: Not all attributes are being forwarded

### DISCUSSION THREAD

**Discussion by wrabit** [laravel/framework#35227] [General]
I have a simple component:

`<x-component something="new" />`

Inside component.blade.php I have:

`<x-another some="other" {{ $attributes }} />`

Inside another.blade.php I have:

`{{ $attributes }}`

What I expect to output: 

`some="other" something="new"`

Instead:

`something="new"`

Is this what you would expect? Is it documented somewhere?

**Comment by rodrigopedra** [comment]
I guess you could "merge" other attributes

https://laravel.com/docs/8.x/blade#default-merged-attributes

~~~html
<x-another {{ $attributes->merge(['some' => 'other']) }}></x-another>
~~~
